docs/services/vision/detection.md

```diff
@@ -254,9 +276,8 @@ my_detector = VisionClient.from_robot(robot, "my_detector")
 
 detections = await my_detector.get_detections_from_camera(cam1)
 
-# If you need to get an image first and then run detections on it,
-# you can do it this way (generally slower but useful if you need to
-# use the image afterwards):
+# If you need to use the image based on the detections, get the image first
+# and then run detections on it. This process is slower.
```

Comment:
```suggestion
# and then run detections on it. This process is slower:
```
I think the colon helps make clear that we are talking about the process below, as opposed to a process not shown that the user would have to write themselves.